common/djangoapps/third_party_auth/models.py

```diff
@@ -332,6 +332,7 @@ class OAuth2ProviderConfig(ProviderConfig):
 
     .. no_pii:
     """
+    KEY_FIELDS = ('backend_name',)
```

Comment:
the thing to consider is if multiple oauth2 providers will ever have the same backend. this will depend on what comes out of the work the team is doing regarding enabling support for oauth2 idps.

Reply:
I think that is a fair assumption to make. I don't think it's a stretch for two providers being self hosted with the same back end. I'll amend this to solve that problem as well. Thanks!